### ISSUE TITLE
Request: Option for Sass / LESS

### ISSUE THREAD

**Issue by salbaldovinos** [assemble/generator-assemble#48]
This would be a secondary request in conjunction with my first request about creating a Foundation Assemble version.

Since you're using Bootstrap (default) and when you build out the Foundation options; it would be awesome to have the option for Sass in the SRC folders that then compile to the DIST.

**Comment by jonschlinkert** [comment]
:+1: sounds like a good idea. we'll make this happen too!

**Comment by salbaldovinos** [comment]
Excellent!

**Comment by caraya** [comment]
Has there been any progress on this? is there anything we can help with? Last activity was over a year ago and I wonder if this project is still alive.

**Comment by jonschlinkert** [comment]
> I wonder if this project is still alive.

we haven't decided. We have an alternative project generator system called `generate` in the works. Initially the plan was to update this generator and also release a generator for `generate` so that users could choose which one they wanted to use (yeoman or generate). But now that we've been testing `generate`, yeoman is comparatively much slower, the API is hard to work with, and testing is harder than it needs to be, so it just doesn't seem worth it. 

Once we get `generate` launched I'll update this issue, and if anyone wants to update this generator and maintain it, that would be great. But it probably won't be @doowb or I.

**Comment by caraya** [comment]
@jonschlinkert is there a repository for Generate?

**Comment by jonschlinkert** [comment]
yes but the code that's pushed up is not usable yet. we refactored most of it again and will be pushing up the code probably this weekend. I'll link back when I do

**Comment by caraya** [comment]
Thank you :+1:

**Comment by jonschlinkert** [comment]
no problem, thanks for the question
